bda602_hw1/hw1.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+import plotly
+import numpy as np
+import sklearn
+import plotly.express as px
+
+
+#r"/mnt/d/Data/CSV_file.csv"
+datafile ="http://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data"
+#c=pd.read_csv(datafile)
+#datafile2 = "/mnt/C:/Users/thoma/OneDrive/Documents/bda602/hw1/bezdekIris.data"
+#print(datafile)
+
+iris_data = pd.read_csv(datafile, sep=",")
+#print(iris_data)
+iris_data.columns =['Sepal_Length', 'Sepal_width', 'petal_length', 'petal_width','species']
+
+print(iris_data.head())
+
+# def petal_func(columnname):
+#     column = iris_data[columnname]
+#     statement = "Column {element} statistics"
+#     print(statement.format(element=columnname))
+#     print(str(np.mean(column)) + " Mean of column")
+#     print(str(np.median(column)) + " Median of Column")
+#     print(str(np.std(column)) + " Standard Deviation of Column")
+#
+#     return column
+#
+#
+# petal_func("5.1")
+# petal_func("3.5")
+# petal_func("1.4")
+# petal_func("0.2")
+
+
+
+# fig = px.histogram(iris_data, x="species", y = "Sepal_Length")
+# fig.show()
+
+# fig2 = px.scatter(iris_data, x="species", y = "Sepal_Length")
+# fig2.show()
+
+# fig3 = px.pie(iris_data, values = "petal_width", names ="species")
+# fig3.show()
+#
+# fig4 = px.line(iris_data, x="species", y = "Sepal_width")
+# fig4.show()
+# #
+# fig5 = px.bar(iris_data, x="species", y = "petal_length")
+# fig5.show()
+
+# Analyze and build models - Use scikit-learn
+# Use the StandardScaler transformer
+# Fit the transformed data against random forest classifier (try other classifiers)
+
+# y = species
+
+
+from sklearn.preprocessing import StandardScaler

```

Comment:
I think it's better to add all the import commands at the beginning of the code, in one place